doc/chapter-firststeps.asciidoc

```diff
@@ -143,15 +145,15 @@ Each link in the article has a number next to it.
 You can now:
 
 * Press any number to open an article link in the browser.
-  For numbers larger than 9 type `#`, then the number and press `Enter`.
-* Press `o` to open the article in the browser.
-* Press `q` to go back to the _Article List View_.
+  For numbers larger than 9 type kbd:[#], then the number and press kdb:[Enter].
```

Comment:
```suggestion
  For numbers larger than 9 type kbd:[#], then the number and press kbd:[Enter].
```

Reply:
Thanks, fixed (amended and force-pushed)